simple-social-icons.php

```diff
@@ -425,55 +433,60 @@ function update( $newinstance, $oldinstance ) {
 	 *
 	 * Outputs the actual widget on the front-end based on the widget options the user selected.
 	 *
+	 * @param array $args The display args.
+	 * @param array $instance The instance settings.
 	 */
-	function widget( $args, $instance ) {
-
-		extract( $args );
+	public function widget( $args, $instance ) {
 
 		/** Merge with defaults */
 		$instance = wp_parse_args( (array) $instance, $this->defaults );
 
-		echo $before_widget;
+		echo $args['before_widget']; // phpcs:ignore
 
-			if ( ! empty( $instance['title'] ) )
-				echo $before_title . apply_filters( 'widget_title', $instance['title'], $instance, $this->id_base ) . $after_title;
+		if ( ! empty( $instance['title'] ) ) {
+			echo $args['before_title'] . apply_filters( 'widget_title', $instance['title'], $instance, $this->id_base ) . $args['after_title']; // phpcs:ignore
```

Comment:
`$args['before_title']` [isn't escaped](https://github.com/WordPress/wordpress-develop/search?q=%24args%5B%27before_title%27%5D) in the Core widgets